### ISSUE TITLE
VSCode Window not showing despite process running (C# project, Unity)

### ISSUE THREAD

**Issue by Ivanca** [dotnet/vscode-csharp#3364]
Issue Type: <b>Bug</b>

Sometimes trying to open a C# file that belongs to a project does nothing, when I open the "process explorer" I can see that the process exists but is not being rendered (the window is nowhere to be found, doesn't appear on mouse over taskbar or alt-tab, or anywhere). In the attached screenshot you can see that the "process explorer" is claiming that 2 windows exist but is only showing one at all.

The only workaround I found is to force-kill the (hidden) window process from the "process explorer" and then reopening the project.

![windowInvisible](https://user-images.githubusercontent.com/907138/67154910-4f0b4e00-f2c9-11e9-80d3-a7161b0e5738.png)

VS Code version: Code 1.39.2 (6ab598523be7a800d7f3eb4d92d7ab9a66069390, 2019-10-15T15:35:18.241Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz (4 x 3499)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|11.95GB (7.68GB free)|
|Process Argv|D:/Dropbox/Laserlogic -g D:\Dropbox\Laserlogic\Assets\ErbGameArt\3D Lasers Pack\Demo scene lasers\EGA_Laser.cs:1:0|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2019.10.41019
remote-wsl|ms-|0.39.9
csharp|ms-|1.21.5
sublime-keybindings|ms-|4.0.3
vetur|oct|0.22.4

</details>

**Comment by ddulshan** [comment]
I am having this issue aswell. What I noticed is that this happens when my Unity Project is created with a specific name, in my case "RPG". I've tested with other names and they all work fine.

**Comment by Ivanca** [comment]
Clicking "Reopen" makes the process re-start with the exact same problem (so I have to click "close" then open it again)
![image](https://user-images.githubusercontent.com/907138/69017020-cb995700-0971-11ea-8319-626240ae761f.png)

**Comment by Ivanca** [comment]
@DustinCampbell @akshita31 @gregg-miskelly @savpek @TheRealPiotrP ☎️

**Comment by Ivanca** [comment]
@DustinCampbell @akshita31 @gregg-miskelly @savpek @TheRealPiotrP
🙏

**Comment by JoeRobich** [comment]
@Ivanca Are you using Window's virtual desktops? Is VS Code open on a different desktop than the one that is active?

**Comment by Ivanca** [comment]
No virtual desktops

> @Ivanca Are you using Window's virtual desktops? Is VS Code open on a different desktop than the one that is active?

No virtual desktop, single desktop/display conf.

**Comment by davidrabineau** [comment]
I am running into the exact same issue today. 

Worth noting that on VS Code's advice, I installed dotnet-sdk-3.1.201 earlier today. I have no idea if that's actually related but it's the only notable change in the environment.
I've tried uninstalling it but no cigar.

Also worth noting that I reinstalled VSCode, when clicking "Run VsCode" at the end of the install process, it opened it with its normal behavior (and displaying the projects i've been trying to open).

I've closed the Vscode window and I'm stuck again in the initial situation. At least thanks to @Ivanca 's workaround I can access my project again, thanks!
